medusa/filtering.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 Spotify AB. All rights reserved.
```

Comment:
@michaelsembwever, which copyright should be added here?
Definitely not the Spotify one, but I'm wondering what's our approach for external contributions.

I like the approach described [here](https://www.linuxfoundation.org/blog/2020/01/copyright-notices-in-open-source-software-projects/), under "Community best practices". 
wdyt?